include/arch/arc/v2/irq.h

```diff
@@ -40,6 +40,21 @@ extern void z_irq_priority_set(unsigned int irq, unsigned int prio,
 extern void _isr_wrapper(void);
 extern void z_irq_spurious(const void *unused);
 
+
+struct arc_isr_sratic_data {
```

Comment:
typo sratic -> static

Reply:
fixed (we don't have this structure in updated version)